docs/FAQ.md

```diff
@@ -0,0 +1,36 @@
+# Frequently Asked Questions (FAQ)
+
+
+## APNs
+
+When iOS notifications do not work, double-check APNS configuration.
+Mainly the host, the key, and the bundle.
+
+APNs expose two hosts, production `api.push.apple.com` and development `api.sandbox.push.apple.com`.
```

Comment:
This is not configurable by the user, so do not disclose it can lead to more misunderstanding